mptt/models.py

```diff
@@ -182,6 +182,11 @@ def insertion_target_filters(self, instance, order_insertion_by):
                 field = instance._meta.get_field(field_name)
                 value = field.pre_save(instance, True)
 
+            # not really sure how this works, but fixes #340 
+            if value is None:
+                value = bool(filter_suffix=='__lt')
```

Comment:
Good question. Thanks for researching that. I _think_ that's the right approach - it doesn't matter in most circumstances, but doing it this way means that if you switch between oracle and another database you don't need to rebuild your tree 👍